src/components/font-pair-picker/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Internal dependencies
+ */
+import { BaseControl, Button } from '~stackable/components'
+
+/**
+ * External dependencies
+ */
+import { i18n } from 'stackable'
+import classNames from 'classnames'
+import { loadGoogleFont } from '~stackable/util'
+import { omit } from 'lodash'
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n'
+
+const FontPairPicker = props => {
+	const headingStyles = props.fontPair.typography.h1
+	const paragraphStyles = props.fontPair.typography.p
+	if ( headingStyles.fontFamily ) {
+		loadGoogleFont( headingStyles.fontFamily )
+	}
+	if ( paragraphStyles.fontFamily ) {
+		loadGoogleFont( paragraphStyles.fontFamily )
+	}
+
+	const label = (
+		<div>
+			<span
+				style={ omit( { ...headingStyles }, [ 'fontSize', 'lineHeight' ] ) }
+				className="ugb-global-settings-font-pair__label"
+			>
+				{ headingStyles.fontFamily ? headingStyles.fontFamily : 'Theme Heading Default' }
```

Comment:
This is too long, please use "Default Heading" and "Default Body" instead.

Also, please go through all the changes, all text strings should be translatable:

```suggestion
				{ headingStyles.fontFamily ? headingStyles.fontFamily : __( 'Default Heading', i18n ) }
```